idle-rpg/utils/enumHelper.js

```diff
@@ -11,6 +11,12 @@ const enumHelper = {
     'dnd'
   ],
 
+  pmMode: {
+    on: 'on',
+    off: 'off',
+    filtered: 'filtered'
+  },
+
```

Comment:
I didn't really know, where to put it.
Should it be higher/lower?

Reply:
And another name maybe?

Reply:
> And another name maybe?

The name seems okay, we can change the naming soon when the reset is ready. Although for now I think its good. Also the place is good, just throw the new ones in the end of the enumhelper as long as we can find it in the helper since a simple ctrl+f can find it or jumping to the enum with ctrl+click within the IDE (If ur IDE supports it)